include/glow/Quantization/Base/Base.h

```diff
@@ -107,6 +108,13 @@ TensorQuantizationParams chooseQuantizationParams(float min, float max,
 std::vector<int8_t> createMapping(TypeRef inTy, TypeRef outTy,
                                   std::function<float(float)> f);
 
+/// Row-wise quantize the tensor \p input. The param \p input is a 2D
+/// tensor (i.e. M * N), \p scales and \p offsets are generated by each row of
+/// \p input, \p qData is 2D tensor quantized from \p input using \p scales and
+/// \p offsets for each row.
+void rowwiseQuantization(Tensor *input, Tensor *qData, Tensor *scales,
```

Comment:
consider change the name to better reflect intent, like rowwiseTensorTransform or something.

Reply:
as we expect all Tensors to be not null, I'd do Tensor & here (with const when needed).

Reply:
@rdzhabarov The input tensor is const. However, if I use const here, "getHandle" can't be used since "'this' argument has type 'const glow::Tensor', but method is not marked const template <class ElemTy> Handle<ElemTy> Tensor::getHandle()" and the code will be complicated. So I didn't put "const" here.